src/UI/examples/Table/Presentation/base.php

```diff
@@ -0,0 +1,64 @@
+<?php
+function base() {
+	global $DIC;
+	$f = $DIC->ui()->factory();
+	$renderer = $DIC->ui()->renderer();
+
+	//build viewcontrols
+	$actions = array ("All" => "#",	"Upcoming events" => "#");
+	$aria_label = "filter entries";
+	$view_controls = array(
+		$f->viewControl()->mode($actions, $aria_label)->withActive("All")
+	);
+
+	//build table
+	$ptable = $f->table()->presentation(
+		'Presentation Table', //title
+		array($view_controls),
+		function ($row, $record, $ui_factory, $environment) { //mapping-closure
+			return $row
+				->withTitle($record['title'])
+				->withSubTitle($record['type'])
+				->withImportantFields(
+					array(
+						$record['begin_date'] => '',
+						$record['location'] => '',
+						$record['bookings_available'] => 'Available Slots: '
```

Comment:
I generally agree, but having two entries with empy key will overwrite the first one...